code/modules/clothing/under/accessories/storage.dm

```diff
@@ -5,6 +5,7 @@
 	_color = "webbing"
 	var/slots = 3
 	var/list/can_only_hold = list() //I would add the other storage item variables, but nothing would use them yet, so there's no point.
+	var/list/cant_hold = list("/obj/item/clothing/accessory/storage", "/obj/item/clothing/under") //NO RECURSION
```

Comment:
Notice how the types in the list are strings.
If they start with =, it does a strict type check.
`...weapons\storage\storage.dm`, lines 253-263